### ISSUE TITLE
Music app crashes in some cases

### ISSUE THREAD

**Issue by bitboy85** [osmandapp/OsmAnd#3682]
I'm using my smartphone for navigation and usually in parallel to play music (there is no usb on my car radio) i noticed that in few cases the music app crashes (or closes) if osmand wants to play sound.
Cases i noticed:
- route recalculation
- using the plugin audio / video notes (if recording a video during navigation)
in other cases like navigation notes everything seems to work fine.

i tested two player apps (Folder player and music folder player) both with the same issue.
The device is rooted, using cyanogenmod 12.1 android 5.1.1
if no one else has this issue please feel free to close it i think its possible that the problem occurs because of cyanogenmod.

**Comment by xmd5a2** [comment]
Any crashlogs?

**Comment by sonora** [comment]
Since the occasions you mention seem the most memory-intensive ones: This may perhaps not be a "crash", could also be Android's built-in behavior to free up memory when needed by closing apps not currently in the foreground (and perhaps the parameterization of this in the cyanogen fw you are using).

**Comment by sonora** [comment]
Chances are this improved now with #12715. **AlthoughI think that our AV notes plugin and our Accessibility plugin may still not follow appropiate sound etiquette**, I only fixed all navigation related sound. (Not sure that could cause other apps to crash, though.)

In any case this was reported for Android 5.1.1, I have implemented some improvements also, while most would affect And 8 and later.

Closing this one, if it still is an issue please open a new issue with more details.
